src/docker-install.md

```diff
@@ -0,0 +1,21 @@
+# Installing HADDOCK3 using docker
```

Comment:
the concept here is wrong - running something with docker is different than installing